packages/shared/src/utils/date/date-time.ts

```diff
@@ -73,7 +73,8 @@ export const toGMTFormat = (time?: moment.MomentInput) =>
         .utc()
         .format('YYYY-MM-DD HH:mm:ss [GMT]');
 
-export const formatDate = (date?: moment.MomentInput, date_format = 'YYYY-MM-DD') => toMoment(date).format(date_format);
+export const formatDate = (date?: moment.MomentInput, date_format = 'YYYY-MM-DD', should_format_null = true) =>
+    !should_format_null && date === null ? undefined : toMoment(date).format(date_format);
```

Comment:
is there any way to avoid `should_format_null` flag? Can we just return undefined if the date is null without checking the flag?

Reply:
This will break current behaviour. At the moment, when you pass it null, moment library returns today's date, therefore, I only made it an optional value.